.github/workflows/py-ci.yml

```diff
@@ -72,6 +72,8 @@ jobs:
         run: python -m pip install poetry
       - name: install deps
         run: cd testdocs && poetry install
+      - name: skip aws test since no credentials present (and side effects unclear)
+        run: rm docs/advanced-features/athena.md
```

Comment:
I still don't quite understand this. Can't we just add `<!--pytest-codeblocks:skipfile-->` to the file?

Reply:
No, because that would also prevent it from running locally.  Yes, this is a hack, but given the risk of running AWS commands in CI it seems expedient.

Reply:
OK